src/axolotl/utils/data.py

```diff
@@ -74,6 +74,11 @@ def prepare_dataset(cfg, tokenizer):
         if isinstance(cfg.pretraining_dataset, dict):
             path = cfg.pretraining_dataset["path"]
             name = cfg.pretraining_dataset["name"]
+        elif isinstance(cfg.pretraining_dataset, list) and isinstance(
```

Comment:
I wonder if there’s a better way to handle this.. can we somehow internally map it to use the same format as “dataset:”, just for loading